### ISSUE TITLE
Vega is missing from picca

### ISSUE THREAD

**Issue by iprafols** [igmhub/picca#886]
We are now using Vega as our main fitter, and fitter2 is deprecated. However, Vega is still not included as part of the package. 
@andreicuceu has some ideas on how to properly add it so that Vega is also automatically installed when installing picca

**Comment by andreufont** [comment]
@andreicuceu , if this turned out to be a pain, we could also leave Vega outside of Picca. It would be nice to have them together, but I'm not sure this is a requirement.

**Comment by andreicuceu** [comment]
@andreufont, I'm still considering the different options. I recently realized that picca is in PyPI (https://pypi.org/project/picca/), so one of the easier approaches would be to also add vega to PyPI, so you can just do "pip install vega". In that case, we could just make vega a dependency of picca, and it would be automatically installed.

There are other options I'm considering, but those would require some small modifications in how we install picca / stuff in picca. We can have a chat about it at the meeting next week.

**Comment by andreufont** [comment]
@iprafols , @andreicuceu - we can close this issue, right?

**Comment by iprafols** [comment]
Yes, I guess we can. We have started the picca_mastermind code that does solve this issue, but we said to wait until we have the final versions for year3 before we start the migration of the delta_extraction part of the code
